go.mod

```diff
@@ -0,0 +1,10 @@
+module kwoc20-backend
+
+go 1.13
+
+require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/gorilla/mux v1.7.4
+	github.com/jinzhu/gorm v1.9.14
+	github.com/kossiitkgp/kwoc20-backend v0.0.0-20200624212038-24d447b5df9c
```

Comment:
We should create github tags for the kwoc20-backend, so that those clumsy commit signatures wouldn't be used. More info - https://blog.golang.org/publishing-go-modules

Reply:
This is great, will look into.
But the last line was added accidentally when made an unnecessary import of `"github.com/kossiitkgp/kwoc20-backend"`, which I removed later. :stuck_out_tongue: